src/cpu/pred/BranchPredictor.py

```
@@ -1057,7 +1057,7 @@ class BTBTAGE(TimedBaseBTBPredictor):
     useAltOnNaSize = Param.Unsigned(128, "Size of the useAltOnNa table")
     useAltOnNaWidth = Param.Unsigned(7, "Width of the useAltOnNa table")
     numBanks = Param.Unsigned(4, "Number of banks for bank conflict simulation")
-    enableBankConflict = Param.Bool(True, "Enable bank conflict simulation")
+    enableBankConflict = Param.Bool(False, "Enable bank conflict simulation")
```

Ruff (0.15.2)

[error] `Param` may be undefined, or defined from star imports

(F405)